app/views/shopping_lists/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<% if user_signed_in? %>
+<h1 class="text-center mb-5">Shopping List</h1>
+
+<div class="d-flex flex-row justify-content-evenly mb-5">
+  <h3>Amount of food items to buy: <%= @recipe_foods.length %> </h3>
+  <h3>Total price: <%= @total_price %> </h3>
+</div>
+
+<table class="table table-bordered table-striped recipe-table">
+    <thead>
+        <tr>
+        <th scope="col">Food</th>
+        <th scope="col">Quantity</th>
+        <th scope="col">Price</th>
+        </tr>
+    </thead>
+    <tbody>
+      <% @recipe_foods.each do |r| %>
+        <% if @food.include?(r.food) %>
+          <tr>
+            <th scope="row"><%= r.food.name %></th>
+            <td><%= r.quantity %></td>
+            <td><%= r.food.Price * r.quantity %></td>
+          </tr>
+        <% end %>
+      <% end %>
+    </tbody>
+    </table>
```

Comment:
- [ ] Kindly add the measuring unit to the food quantity and also add currency value e.g `$` to the calculated value and price.

Your Implementation | Wireframe
:---:|:---:
![](https://i.imgur.com/7yb4SDs.png) | ![](https://i.imgur.com/kB5EPhh.png)

Reply:
Thanks for your feedback. We fixed the issue.